document/js-api/index.bs

```diff
@@ -286,22 +286,22 @@ A {{Module}} object represents a single WebAssembly module. Each {{Module}} obje
 </div>
 
 <div algorithm>
-  To <dfn>asynchronously compile a WebAssembly module</dfn> from a {{BufferSource}} |bytes|, with the promise |promise|, using optional [=task source=] |taskSource|, perform the following steps:
+  To <dfn>asynchronously compile a WebAssembly module</dfn> from a {{BufferSource}} |bytes|, using optional [=task source=] |taskSource|, perform the following steps:
```

Comment:
Note that this algorithm is also used from https://webassembly.github.io/spec/web-api/index.html#compile-a-potential-webassembly-response

Reply:
Oops, right, will fix this!

Reply:
Fixed